### DISCUSSION TITLE
Asset Inventory: Capturing perils to which each asset is exposed - Factors in the asset’s location (eg, “located on coast”, or “located in landslide zone”) that pose a risk in the event of a hazard – C/T

### DISCUSSION THREAD

**Discussion by sreyac** [CriticalAssetManagement/CAMS#122] [General]
Require further clarification and discussion

**Comment by sreyac** [comment]
Peter will give a checklist for this

**Comment by DAKJ22** [comment]
As a follow-up to our call see: 
https://cdbgdr.vihfa.gov/wp-content/uploads/2021/07/USVI-HUD-Approved-Mitgation-Action-Plan-7142021.pdf
